Modules/CraftQueue/CraftQueue.lua

```diff
@@ -705,6 +707,33 @@ function CraftSim.CRAFTQ:GetRestockOptionsForRecipe(recipeID)
     return restockPerRecipeOptions[recipeID]
 end
 
+-- Function to determine the restock quantity for a given expected item.
+-- 
+-- This function retrieves the default restock amount from the database options.
+-- If a custom restock key for TSM (TradeSkillMaster) is specified, it attempts to get the restock amount
+-- using the TSM API based on the expected item's link. If a valid restock amount is found, it returns that value.
+-- Otherwise, it falls back to the default restock amount.
+--
+---@param expectedItem ItemMixin The item for which the restock quantity is being determined.
+---@return number restockAmount Restock quantity for the given expected item.
+function CraftSim.CRAFTQ:GetRestockQuantity(expectedItem)
+    local defaultRestockAmount = CraftSim.DB.OPTIONS:Get("CRAFTQUEUE_GENERAL_RESTOCK_RESTOCK_AMOUNT") or 1
+    if expectedItem and CraftSim.DB.OPTIONS:Get("TSM_RESTOCK_KEY_ITEMS") ~= "" then
+        if not TSM_API then
+            print("Error: TSM_API is not defined.")

```

Comment:
You can remove the print as it should be default behaviour is users e.g. only use Auctionator and Craftsim
Maybe add to the condition above like
`if TSM_API and expectedItem and CraftSim.DB.OPTIONS:Get("TSM_RESTOCK_KEY_ITEMS") ~= ""`